### PR TITLE
FOIA-000: Update docs for application entry point, page, and data sto…

### PR DESCRIPTION
**Pull request by pixlkat**
…res.

### CODE REVIEW

File: docs/annual-report-data-app.md [hunk 1]
```diff
@@ -12,24 +12,102 @@ The app uses the same architecture as the [Request app](request-app.md).
...
+the history object to be passed to it as a prop.
+```
+export default withRouter(Container.create(AnnualReportDataPage, {
+  withProps: true,
+}));
+```
+
+Usage in application entry point:
+```
+...
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you remove these ...

File: docs/annual-report-data-app.md [hunk 2]
```diff
@@ -12,24 +12,102 @@ The app uses the same architecture as the [Request app](request-app.md).
...
 
+The following data stores are used by the app's Container to maintain state. A brief description of each store and
+its purpose follows.
+
 `/js/stores/annual_report_data_form.js` 
 
-The `AnnualReportDataFormStore` contains the form state for the annual report
-that will be requested. It contains a separate section for each report section
-on the form the user sees.
+The `AnnualReportDataFormStore` contains the form state for the annual report that will be requested.  The state
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you ensure the character length per line does not exceed 80?

File: docs/annual-report-data-app.md [hunk 3]
```diff
@@ -12,24 +12,102 @@ The app uses the same architecture as the [Request app](request-app.md).
...
+to populate the Fiscal Year checkbox list on the form.
+```
+this.state = {
+  fiscalYears: new List(),
+};
+```
+
+`/js/stores/agency_component.js`
+
+The `AgencyComponentStore` is used to populate the available agencies for the agency selector. This is a shared data
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you ensure the character length per line does not exceed 80?

File: docs/annual-report-data-app.md [hunk 4]
```diff
@@ -12,24 +12,102 @@ The app uses the same architecture as the [Request app](request-app.md).
...
+  fiscalYearsDisplayError: false,
+  dataTypeDisplayError: false,
+  agencyComponentDisplayError: false,
+  submissionAction: false,
+};
+```
+
+`js/stores/annual_report_fiscal_year.js`
+
+The 'AnnualReportFiscalYearStore' contains the list of fiscal years received from the API backend. This list is used
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you ensure the character length per line does not exceed 80?

File: docs/annual-report-data-app.md [hunk 5]
```diff
@@ -12,24 +12,102 @@ The app uses the same architecture as the [Request app](request-app.md).
...
+  agencies: new Map(),
+  agencyComponents: new List(),
+  agencyFinderDataComplete: false,
+  agencyFinderDataProgress: 0,
+};
+```
+
+`/js/annual_report_data_types.js`
+
+The `AnnualReportDataTypesStore` contains the data types which can be selected in Section 2 of the report with the
```
Comment (bemarlan) [inline docs/annual-report-data-app.md]: Can you ensure the character length per line does not exceed 80?

**Comment by pixlkat** [comment]
@bemarlan changes (and rebase) made and pushed.

**Pull request merged**